internal/platform/cpuid_amd64.go

```diff
@@ -48,12 +48,15 @@ func loadCpuFeatureFlags() CpuFeatureFlags {
 	}
 }
 
-// Has implements the same method on the CpuFeatureFlags interface
+// Has implements the same method on the CpuFeatureFlags interface.
 func (f *cpuFeatureFlags) Has(cpuFeature CpuFeature) bool {
 	return (f.flags & uint64(cpuFeature)) != 0
 }
 
-// HasExtra implements the same method on the CpuFeatureFlags interface
+// HasExtra implements the same method on the CpuFeatureFlags interface.
 func (f *cpuFeatureFlags) HasExtra(cpuFeature CpuFeature) bool {
 	return (f.extraFlags & uint64(cpuFeature)) != 0
 }
+
+// Raw implements the same method on the CpuFeatureFlags interface.
+func (f *cpuFeatureFlags) Raw() [2]uint64 { return [2]uint64{f.flags, f.extraFlags} }
```

Comment:
I wonder if it's too much effort to enumerate `CpuFeature` and return only those bits. Otherwise, it will be quite hard to have offline compilation work in practical environments, any architectural difference between CI and server machine would be incompatible (I think?). Of course, not crashing is more important so this would still be a fine change vs not.

Reply:
I think I would like to have the input from @evacchi

Reply:
https://github.com/tetratelabs/wazero/pull/2220/commits/3de5bcb59d0bf62a2ef295b42fb866843bbc8612 you cool with this?

Reply:
looks good!